hasher-matcher-actioner/src/OpenMediaMatch/blueprints/hashing.py

```diff
@@ -124,11 +124,22 @@ def hash_media() -> dict[str, str]:
 
     try:
         # Get response with content length tracking
-        max_content_length = int(
-            current_app.config.get("MAX_CONTENT_LENGTH", DEFAULT_MAX_CONTENT_LENGTH)
-        )
+        max_file_size = current_app.config.get("MAX_REMOTE_FILE_SIZE")
+        if max_file_size is None:
+            max_file_size = DEFAULT_MAX_REMOTE_FILE_SIZE
+
+        # cast to integer if necessary (the value could have come from an environment variable)
+        if isinstance(max_file_size, str):
+            if not max_file_size.isdigit():
+                logger.error(
+                    f"MAX_REMOTE_FILE_SIZE misconfigured, expected integer, received: {max_file_size}"
+                )
+                abort(500, "Service misconfigured, see logs for details")
+
+            max_file_size = int(max_file_size)
```

Comment:
Interestingly, this wouldn't be necessary with fastapi, where you have a "configuration class" that does all the parsing and validation of configuration before starting the app.

Reply:
Yeah, the comment was more meant to say that "maybe there's a way to do similar configuration loading with Flask, where environment variables / config are validated as they're loaded to be a specific type.